Command/RemoveThumbsCommand.php

```diff
@@ -236,4 +257,27 @@ protected function log($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * @return QuestionHelper|DialogHelper
+     */
+    private function getQuestionHelper()
+    {
+        // NEXT_MAJOR: Remove this BC code for SensioGeneratorBundle 2.3/2.4 after dropping support for Symfony 2.3
+        if (class_exists('Sensio\Bundle\GeneratorBundle\Command\Helper\DialogHelper')) {
+            $questionHelper = $this->getHelper('dialog');
+            if (!$questionHelper instanceof DialogHelper) {
```

Comment:
@Soullivaneuh I think the usage of instanceof is **not** enough, because we're not only checking the variable by `instaceof`, but we're also creating `new DialogHelper()`.

So simplified code like this will probably cause a bug:
```php
    /**
     * @return QuestionHelper|DialogHelper
     */
    private function getQuestionHelper()
    {
        // NEXT_MAJOR: Remove this BC code for SensioGeneratorBundle 2.3/2.4 after dropping support for Symfony 2.3
        if (!($questionHelper = $this->getHelper('dialog')) instanceof DialogHelper) {
            $questionHelper = new DialogHelper();
            $this->getHelperSet()->set($questionHelper);
        } elseif (!($questionHelper = $this->getHelper('question')) instanceof QuestionHelper) {
            $questionHelper = new QuestionHelper();
            $this->getHelperSet()->set($questionHelper);
        }

        return $questionHelper;
    }
```
because `!($questionHelper = $this->getHelper('dialog')) instanceof DialogHelper` will return `true` even if the class `Sensio\Bundle\GeneratorBundle\Command\Helper\DialogHelper` does not exist — and in the next line we're trying to create new instance of not existing class.